frontend/src/test/scala/bloop/exec/ProcessConfigSpec.scala

```diff
@@ -0,0 +1,72 @@
+package bloop.exec
+
+import org.junit.Test
+import org.junit.Assert.{assertEquals, assertNotEquals}
+
+import bloop.logging.RecordingLogger
+import bloop.tasks.ProjectHelpers
+
+class ProcessConfigSpec {
+
+  object ArtificialSources {
+    val `A.scala` = s"""package $packageName
+                       |object $mainClassName {
+                       |  def main(args: Array[String]): Unit = {
+                       |    if (args.contains("crash")) throw new Exception
+                       |    println(s"Arguments: $${args.mkString(", ")}")
+                       |    System.err.println("testing stderr")
+                       |  }
+                       |}""".stripMargin
+  }
+
+  val packageName = "foo.bar"
```

Comment:
If this is being used by the previous object, why not have it defined before `ArtificialSources`? If `ArtificialSources` weren't an object, it will be a null pointer exception or a wrong forward reference. It's better to protect ourselves from doing the wrong thing in the future should that change.